.azure/pipelines/production/Phonebook.Assets.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - bash: |
           docker build -t $dockerRepo:$tag .
-          docker login -u $DOCKER_REGISTRY_USER -p $(DOCKER_REGISTRY_PASSWORD)
+          docker login -u $DOCKER_REGISTRY_USER -p $DOCKER_REGISTRY_PASSWORD
```

Comment:
why does this work. Build variables are always in $(variablename)?

Reply:
no, they are environment variables and recently azure has a problem parsing your syntax...

Reply:
https://dev.azure.com/T-Systems-MMS/phonebook/_build/results?buildId=644&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774